docs/about-us.rst

```diff
@@ -0,0 +1,61 @@
+.. SPDX-FileCopyrightText: 2023 The meson-python developers
+..
+.. SPDX-License-Identifier: MIT
+
+.. _about-us:
+
+********
+About Us
+********
+
+``meson-python`` is a project that extends the Meson_ build system, making it
+possible to it for Python_ packaging. It is 100% open source and is available
+under the permissive `MIT (Expat) license`_. The project was started in 2021,
```

Comment:
please remove (Expat), it's known as the MIT license

Reply:
Just stating MIT is ambiguous, we are using the Expat variant of the MIT license, and that's why I added it. I am pretty much indifferent, so I'll make the change unless someone else has an opinion.

Reply:
What? Our SPDX headers state that the license is MIT, not "MIT (Expat)". There is no "MIT (Expat)" license in the SPDX list of licenses https://spdx.org/licenses/ or in the OSI approved licenses https://opensource.org/licenses/

Reply:
SPDX defines a canonical text for each identifier, that's why it isn't mentioned there. They do define several other variants of the MIT license.

From https://www.gnu.org/licenses/license-list.html#Expat
> Some people call this license “the MIT License,” but that term is misleading, since MIT has used many licenses for software. It is also ambiguous, since the same people also call the [X11 license](https://www.gnu.org/licenses/license-list.html#X11License) “the MIT License,” failing to distinguish them. We recommend not using the term “MIT License.”

But anyway, I really don't care much, so I will remove it.